sites/gent/ch_hpc_policies.tex

```diff
@@ -1,4 +1,85 @@
 \chapter{HPC Policies}
 \label{ch:hpc-policies}
 
-Stub chapter
+Everyone can get access and use the HPC-UGent supercomputing infrastructure and services.
```

Comment:
Hmm, maybe "everyone" is a bit too broad?

What about a researcher from a US university, for example?

Reply:
If they pay, they can get access.

Reply:
Then we need an extra subsection, after Industry, named "Others"?

Reply:
Nah, will complicate too much
When they are in doubt, they should mail period